test/e2e/config/docker.yaml

```diff
@@ -60,7 +60,7 @@ providers:
     value: ../../../config/default
     replacements:
     - old: --metrics-bind-addr=localhost:8080
-      new: --metrics-bind-addr=:8080
+      new: "--metrics-bind-addr=:8080\n        - --logging-format=json"
```

Comment:
TBD if we want to find a better way to add logging-format. Afaik currently the e2e test config doesn't support another way (apart from adding --logging-format to all managers).

Reply:
This isn't nice.
Probably we should introduce manager args (override, additions, possibly removal) so we are not rely too much on string replacements

Reply:
Yup it's definitely one of the worst hacks possible.

I'll open a follow-up issue so we can clean this up

Reply:
Yup it's definitely one of the worst hacks possible.

I'll open a follow-up issue so we can clean this up

Reply:
Yup it's definitely one of the worst hacks possible.

I'll open a follow-up issue so we can clean this up